stem/IO/kratos_solver_io.py

```diff
@@ -0,0 +1,220 @@
+from typing import Dict, Any, List
+from copy import deepcopy
+
+from stem.solver import *
+from stem.model_part import ModelPart, BodyModelPart
+
+
+class KratosSolverIO:
+    """
+    Class containing methods to write Kratos solver settings
+
+    Attributes:
+        - ndim (int): The number of dimensions of the problem (2 or 3).
+        - domain (str): The name of the Kratos domain.
+    """
+    def __init__(self, ndim: int, domain: str):
+        """
+        Class to read and write Kratos solver settings.
+
+        Args:
+            - ndim (int): The number of dimensions of the problem (2 or 3).
+            - domain (str): The name of the Kratos domain.
+        """
+        self.ndim: int = ndim
+        self.domain: str = domain
+
+    @staticmethod
+    def __create_problem_data_dictionary(problem_data: Problem):
+        """
+        Creates a dictionary containing the problem data
+
+        Args:
+            - problem_data (:class:`stem.solver.Problem`): The problem data
+
+        Returns:
+            - Dict[str, Any]: dictionary containing the problem data
+        """
+        problem_data_dict: Dict[str, Any] = {"problem_name": problem_data.problem_name,
+                                             "start_time": problem_data.settings.time_integration.start_time,
+                                             "end_time": problem_data.settings.time_integration.end_time,
+                                             "echo_level": problem_data.echo_level,
+                                             "parallel_type": "OpenMP",
```

Comment:
should this be an option in stem.solver.Problem?

Reply:
other than openMP dont work well, so for now, no.